dist/common/scripts/scylla_io_setup

```diff
@@ -172,12 +193,15 @@ def run_iotune():
                 iotune_args += [ "--smp", str(cpudata.smp()) ]
 
             configure_iotune_open_fd_limit(cpudata.nr_shards())
-
             try:
-                subprocess.check_call([bindir() + "/iotune",
+                cmd_args = [bindir() + "/iotune",
                                        "--format", "envfile",
                                        "--options-file", etcdir() + "/scylla.d/io.conf",
-                                       "--properties-file", etcdir() + "/scylla.d/io_properties.yaml"] + iotune_args)
+                                       "--properties-file", etcdir() + "/scylla.d/io_properties.yaml"]
+                if should_force_random_request_size():
+                    cmd_args += ["--random-write-io-buffer-size", "4096",
+                                 "--random-read-io-buffer-size", "4096"]
+                subprocess.check_call(cmd_args + iotune_args)
```

Comment:
nit: it's enough to update `iotune_atgs` with the options and leave the `subprocess.check_call([bindir() + "/iotune", ...` thing intact